### PR TITLE
External Identifier Search Error

### LINKED ISSUE #18: /extidentifier.json throws 500 error
If you request /extidentifier.json, it throws a 500 error when there is not an ark in the request.  In [coda_mdstore/views.py in externalIdentifierSearchJSON](https://github.com/unt-libraries/coda/blob/master/coda/coda_mdstore/views.py#L675).
```
TypeError at /extidentifier.json
argument of type 'NoneType' is not iterable
```

### PR DESCRIPTION
**Pull request by damonkelley**
Fixes #18 

This 
1. Resolves the 500 error
2. Cleans up the the view a bit
3. Removes the old `TEMPLATE_CONTEXT_PROCESSORS` settings.

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by damonkelley** [comment]
Updating the `settings/base.py` was becoming too large of a task for the scope of this PR. I have removed all the changes to that file except for the adding the messages middleware class.

**Comment by ldko** [comment]
:+1:

File: coda/coda_mdstore/models.py [hunk 1]
```diff
@@ -4,29 +4,32 @@
 class Bag(models.Model):
     name = models.CharField(
         max_length=255,
-        help_text="Name of Bag",
-        primary_key=True
-    )
+        primary_key=True,
+        help_text="Name of Bag")
```
Comment (somexpert) [inline coda/coda_mdstore/models.py]: Not to be picky or anything, but if you're already making purely stylistic changes, something to consider would be changing all the quotes here to single quotes.

Comment (damonkelley) [inline coda/coda_mdstore/models.py]: Sure

**Comment by somexpert** [comment]
:thumbsup:

**Pull request merged**